gapid_tests/command_buffer_tests/vkTrimCommandPool_test/vkTrimCommandPool.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gapit_test_framework import gapit_test, require, require_equal
+from gapit_test_framework import require_not_equal, GapitTest
+from gapit_test_framework import get_read_offset_function, get_write_offset_function
+from vulkan_constants import *
+from struct_offsets import VulkanStruct, UINT32_T, POINTER, HANDLE, DEVICE_SIZE
+from struct_offsets import ARRAY, CHAR
```

Comment:
You can probably remove some imports. For instance, `require_not_equal` does not seem to be used anywhere in this file. Can you try and cleanup the imports?